pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -91,14 +91,11 @@ func (vm *Windows) runBootstrapper() error {
 	}
 	wmcbInitializeCmd := remoteDir + "\\wmcb.exe initialize-kubelet --ignition-file " + winTemp +
 		"worker.ign --kubelet-path " + winTemp + "kubelet.exe"
-	stdout, stderr, err := vm.Run(wmcbInitializeCmd, true)
+	out, err := vm.RunOverSSH(wmcbInitializeCmd, true)
+	log.V(1).Info("output from wmcb", "output", out)
```

Comment:
Why V(1) instead instead of V(5)? Also this line should be after err nil check or else it might be a problem

Reply:
I thought that is the standard @mansikulkarni96 introduced i.e. all debug logs should be `V(1)`. If that is not the case I am fine using `V(5)`.

Reply:
Got it. I am fine with it then.